examples/CRISP/scripts/lib/dev_config.sh

```
@@ -0,0 +1,96 @@
+  _set_env_kv "$server_env" "E3_PROOF_AGGREGATION_ENABLED" "$CRISP_PROOF_AGGREGATION_ENABLED"
+}
+
+compile_enclave_dkg_circuits_if_needed() {
+  if [[ "$CRISP_PROOF_AGGREGATION_ENABLED" != "true" ]]; then
+    echo "Skipping enclave DKG circuit build (CRISP_PROOF_AGGREGATION_ENABLED=false in crisp.dev.env)"
+    return 0
+  fi
+
+  echo "Building enclave DKG circuits (preset=${CRISP_BFV_PRESET})..."
+  (
+    cd "${REPO_ROOT}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)